02.calendar/cal.rb

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+params = ARGV.getopts("y:m:")
+
+year = params["y"].to_i
+month = params["m"].to_i
+
+current_year = Date.today.year
+current_month = Date.today.month
+
+if params["y"].nil?
+  year = current_year
```

Comment:
```suggestion
  year = Date.today.year
```

とすれば`current_year`はなくせますね

Reply:
確かに、無駄ですね！